sandbox/Sandbox.jsx

```diff
@@ -64,6 +64,10 @@ export default class Sandbox extends React.Component {
         // });
     };
 
+    onDoubleClickNode = id => {
+        !this.state.config.collapsible && window.alert(`Double clicked node ${id}`);
```

Comment:
Can you give it a try and write before the call add:

```
event.stopPropagation()
```

Note that first you need to grab the event in the parameters

Reply:
I have done many attempts but it didn't work ... I've also tried preventDefault() but it doesnt do anything ...
I was thinking that maybe there would be a way to say to d3-zoom to zoom only if the double click happens on the graph ? As the onClickGraph function isn't triggered when clicking the node, it consider both clicks differently and we could leverage that ...

Reply:
Currently, the only way I found to solve this event was disabling double-click zoom on the whole graph, by adding .on("dblclick.zoom", null) to the _zoomConfig function. I am trying to implement it only on the nodes but that doesn't work yet